src/python/WMComponent/TaskArchiver/TaskArchiverPoller.py

```diff
@@ -226,33 +222,28 @@ def completeTasks(self, finishedwfs):
         if centralCouchAlive:
             for workflow in finishedwfs:
                 try:
-                    #Notify the WorkQueue, if there is one
+                    # Notify the WorkQueue, if there is one
                     if self.workQueue != None:
                         subList = []
                         logging.info("Marking subscriptions as Done ...")
                         for l in finishedwfs[workflow]["workflows"].values():
                             subList.extend(l)
                         self.notifyWorkQueue(subList)
 
-                    #Now we know the workflow as a whole is gone, we can delete the information from couch
+                    # Tier-0 case, the agent has to mark it completed
                     if not self.useReqMgrForCompletionCheck:
```

Comment:
and this code as well. It's only used by T0, right? since there is no aborted-completed status for T0 requests, I removed this logic as well